libreyolo/models/fomo/dataset.py

```
@@ -0,0 +1,156 @@
+            boxes.append([x1, y1, x2, y2])
+            classes.append(cls)
+        if not boxes:
+            return np.zeros((0, 4), dtype=np.float32), np.zeros((0,), dtype=np.int32)
+        return np.array(boxes, dtype=np.float32), np.array(classes, dtype=np.int32)
+
+    def __getitem__(self, idx: int):
+        img_path = self.img_files[idx]
+        try:
+            pil_img = Image.open(img_path)
+        except (FileNotFoundError, OSError) as e:
+            raise FileNotFoundError(f"Cannot read image: {img_path}") from e
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)